File: DATN_Server/app/Models/Schedule.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;

class Schedule extends Model
{
    use HasFactory;
    protected $fillable = [
        'student_id',
        'teacher_id',
        'phase_id',
        'reportDate',
        'reportContent'
    ];

    protected $primaryKey = 'id';
}

File: DATN_Mobile/app/src/main/java/com/example/qldatn/models/Schedule.kt
package com.example.qldatn.models

class Schedule(val id:Int,
               val student_id:Int,
               val teacher_id:Int,
               val phase_id:Int,
               val reportDate:String,
               val reportContent:String,
               val status:String,
               val refuseContent:String,
               val student_name:String,
              ) {
}